hoodie-client/src/main/java/com/uber/hoodie/HoodieReadClient.java

```diff
@@ -183,4 +187,17 @@ private void assertSqlContext() {
       throws HoodieIndexException {
     return index.tagLocation(hoodieRecords, jsc, hoodieTable);
   }
+
+  /**
+   * Return all pending compactions with instant time for clients to decide what to compact next.
+   * @return
+   */
+  public List<Pair<String, HoodieCompactionPlan>> getPendingCompactions() {
```

Comment:
I'm guessing this will be addressed as part of the next refactor, but a little worried about adding a new API in HoodieWriteClient in a new hoodie release (which we will have to perform to be able to use the build internally) and then take it away from WriteClient. Can we quickly move this to ReadClient which is safer to refactor later ?
Thoughts ? @vinothchandar

Reply:
@n3nash : This is part of HoodieReadClient now.

Reply:
Awesome!